docs/source/release/v6.12.0/Framework/Algorithms/Bugfixes/38290.rst

```diff
@@ -0,0 +1 @@
+- some parameter names in :ref:`algm-HeliumAnalyserEfficiency` were updated for consistency: `GasPressureTimesCellLength` is now `PxD`, `GasPressureTimesCellLengthError` is now `PXDError`, `StartLambda` is now `StartX`, and `EndLambda` is now `EndX`.
```

Comment:
I like the level of detail in this release note. Please could we start the first word with a capital letter, though, and just re-word it slightly to the following:

The following parameter names in :ref:`algm-HeliumAnalyserEfficiency` have been updated for consistency: `GasPressureTimesCellLength` is now `PxD`, `GasPressureTimesCellLengthError` is now `PXDError`, `StartLambda` is now `StartX`, and `EndLambda` is now `EndX`. Any scripts using the old names will need to be updated.

Reply:
Thanks, I have updated the release note with the suggested changes